eth/downloader/downloader.go

```diff
@@ -102,6 +93,7 @@ var (
 	errCancelHeaderProcessing  = errors.New("header processing canceled (requested)")
 	errCancelContentProcessing = errors.New("content processing canceled (requested)")
 	errNoSyncActive            = errors.New("no sync active")
+	errTooOld                  = errors.New("peer doesn't speak recent enough protocol version (need version >= 62)")
```

Comment:
I don't think this is necessary. `eth62`+ is enforced by the devp2p layer, it cannot happen (in geth) that an `eth61` peer finds its way into the downloader.

The only scenario I can imagine this happening is if someone's using our downloader codebase in an extrenal project and has `eth/61` enabled. Was this the case you tried to cover?